### PR TITLE
SDAN-687 Add Atom endpoint to news api

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

**Comment by marwoodandrew** [comment]
@petrjasek @MarkLark86  I've got a couple of PR's here that have been open for a while waiting for a review. I've been patching our production system but probably should do a proper build. I guess at some point I'm going to have to move to the other repo but that will involve an elasticsearch upgrade.

File: newsroom/news_api/news/atom/atom.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+@blueprint.route('/atom', methods=['GET'])
+def get_atom():
+
+    def _format_date(date):
+        iso8601 = date.isoformat()
+        if date.tzinfo:
+            return iso8601
+        return iso8601 + 'Z'
+
+    def _format_date_2(date):
```
Comment (petrjasek) [inline newsroom/news_api/news/atom/atom.py:32]: what does `_2` mean?

File: setup.py [hunk 2]
```diff
@@ -6,16 +6,14 @@
...
     'flask-webpack>=0.1.0,<0.2',
     'Flask-WTF>=0.14.2,<0.15',
     'flask-limiter>=0.9.5.1,<0.9.6',
     'Flask-Cache>=0.13.1,<0.14',
     'flask_pymongo>=0.5.2,<1.0',
     'honcho>=1.0.1',
     'gunicorn>=19.7.1',
-    'superdesk-core>=1.31.1,<=1.32',
-    'icalendar>=4.0.3,<4.1',
+    'superdesk-core>=1.33,<=1.34',
```
Comment (petrjasek) [inline setup.py:16]: I guess this should be rather `<1.34`

**Review by petrjasek — APPROVED** [review]

File: newsroom/news_api/news/atom/atom.py (hunk 1 above)
Comment (marwoodandrew) [reply newsroom/news_api/news/atom/atom.py]: Renamed with a more descriptive name!

File: setup.py (hunk 2 above)
Comment (marwoodandrew) [reply setup.py]: Mark sorted this!

**Pull request merged**